pkg/controller/podautoscaler/horizontal.go

```diff
@@ -274,7 +274,7 @@ func (a *HorizontalController) reconcileKey(key string) error {
 }
 
 // Computes the desired number of replicas for the specified metric of type ObjectMetricSourceType.
-func (a *HorizontalController) computeReplicasForObjectMetric(currentReplicas int32, metricSpec autoscalingv2.MetricSpec, hpa *autoscalingv2.HorizontalPodAutoscaler, selector labels.Selector, status *autoscalingv2.MetricStatus) (int32, time.Time, string, error) {
+func (a *HorizontalController) computeReplicasForObjectMetricSetStatus(currentReplicas int32, metricSpec autoscalingv2.MetricSpec, hpa *autoscalingv2.HorizontalPodAutoscaler, selector labels.Selector, status *autoscalingv2.MetricStatus) (int32, time.Time, string, error) {
```

Comment:
I'd maybe just call this `computeStatusForObjectMetric`  -- I think that's sufficiently descriptive (this name is a bit difficult to parse).

Reply:
Done.